src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
+    pub bytes_per_line: usize,
+    /// Number of bytes per group
```

Comment:
Controls the number of bytes per group, before inserting a ` ` (0x20)

    Default is 1

TODO: Would it be feasible to add also a option for a `byte_group_separator` analogous to `addr_hex_seperator` etc?

Reply:
I don't see why not, seems easy enough to replace the "".repeat for a variable. I don't see it very useful, but if you think it is, it costs nothing to add, I'm happy to do it. Probably in a different pull request I would say, to separate this one which was porting something fixed to be configurable, and the other one will be expanding the functionality to add that other option. But again, happy with what you prefer.

Reply:
Do it in this PR if that's okay for you. If you don't want to do it, I'm fine with it either, then I'll probably do it, but that's totaly fine.